### ISSUE TITLE
repo organization

### ISSUE THREAD

**Issue by dcsan** [sayawan/flowwy#7]
I usually use this structure for apps:

```
/app
/bin
/docs
/packages
```
- app
  this means that the app itself goes one layer down.
- packages
  if separate packages are created these are put in /packages/ and can be sym linked.
  this makes it very easy to create a separate top level app that just uses a couple of the packages in isolation. I find this very useful for building quick tests to isolate problems. 
- docs
  it also keeps any docs or notes, or other files out of the deployed app directory.
  you could put a .idea project here too.
- bin
  any scripts for deploying, setting up server etc. these also should not be in the app dir.

I adopted this in my fork of the repo, however its a big change so it completely breaks git merges... if you adopt this it would help, or i can setup a new cleaner repo.

**Comment by kakajansh** [comment]
It sounds good. 
And liked that structure, too. Adopting it will not be problem for me. But I think if we plan some sort of application for famous demo day, we should setup new repo. And plan about our app structure. Maybe with some sketching, what is app about, functionalities.. 
![sc](https://cloud.githubusercontent.com/assets/7093483/7672083/443928f6-fcee-11e4-8464-170c6d00a0e0.jpg)

**Comment by dcsan** [comment]
sounds good. maybe we can start with a new google doc to outline the main features?

**Comment by dcsan** [comment]
ideally i'd also like to build this mini-framework itself as a package, so that I can use it as a kind of core module in my main app with the router and some core views, and just extend it with other views in the same format perhaps as different packages.  other people can do the same then too.

**Comment by dcsan** [comment]
QuizUp is a pretty nice app too eh. putting together the framework for something like that would be great.

ElevateApp and Peak are two awesome Brain training apps that have some quite simple but elegant design. Especially the Elevate minigames are beautiful to look at (but that's not really framework).

**Comment by kakajansh** [comment]
yes, we can start with outlining main features. And about packaging, it could look something like [raix library](https://github.com/raix/library/tree/master/meteor/core) And including to the project with famono. I am sure there are many other meteorites just waiting  to the famous mixed mode. After its release many things will change.

**Comment by kakajansh** [comment]
Mostly, I think we are already ready for building apps like QuizUp, as @TheAlphaNerd described [here](https://github.com/Famous/famous/issues/692#issuecomment-98848980) 

> The current release is definitely usable. MixedMode will be released at the jQuerySF event.  
>     http://jquerysf.com
